Src/Plugins/AzureDevOpsConfiguration/AZC101.BuildDefinitionSecurity.json

```diff
@@ -0,0 +1,33 @@
+{
+  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
+  "ValidatorsAssemblyName": "AzureDevOpsConfiguration.dll",
+  "Definitions": [
+    {
+      "Id": "AZC101",
+      "Name": "BuildDefinitionShouldBeSecure",
+      "Level": "Error",
+      "Description": "Make sure build definitions in AzureDevOps are secure",
+      "FileNameAllowRegex": "$BuildDefinitionFiles",
+      "MatchExpressions": [
+        {
+          "Id": "AZC101/240",
+          "Name": "BuildDefinitionShouldBeSecure/DontMakeSecretsAvailableToForkedBuilds",
+          "Message": "[Build Definition Id:{0:scanTarget}]({1:targetPath}) in organization '{2:org}' project '{3:project}' was found {4:issueDesc}\r\n{5:description}\r\n{6:recommendation}",
```

Comment:
>{1:targetPath} [](http://example.com/codeflow?start=59&length=14)

As disucssed with Eddy the targetPath expected to be from  ItemPathUri column, e.g.: [https://dev.azure.com/org/project/_apis/build/Definitions/11212?revision=2](https://dev.azure.com/org/project/_apis/build/Definitions/11212?revision=2)